tests/python/opinfo/opinfo_fusion_definitions.py

```diff
@@ -8,11 +8,20 @@
 from opinfo_core import OpInfo
 from opinfo_utils import ArgumentType, is_tensor
 
-from nvfuser import FusionDefinition
-from nvfuser.pytorch_utils import (
-    python_scalar_to_nvfuser_dtype,
-    torch_dtype_to_nvfuser_dtype,
-)
+import sys
+
+if "nvfuser" in sys.modules:
+    from nvfuser import FusionDefinition
+    from nvfuser.pytorch_utils import (
+        python_scalar_to_nvfuser_dtype,
+        torch_dtype_to_nvfuser_dtype,
+    )
+else:
+    from nvfuser_direct import FusionDefinition
+    from nvfuser_direct.pytorch_utils import (
+        python_scalar_to_nvfuser_dtype,
+        torch_dtype_to_nvfuser_dtype,
```

Comment:
this one is sneaky... we are trying to re-use opinfos across direct binding and the old python API.

> Add broadcast to direct bindings to pass binary operations in opinfo tests.

Haha, the PR description was a bit vague about that one.

Reply:
The team wants me to reuses the existing tests instead of duplicating them.

Reply:
Yeah, totally on board with that.